system-tests/services/test-apps.js

```diff
@@ -1212,5 +1217,422 @@ describe("Services", function() {
         .contains("Running", { timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT })
         .should("exist");
     });
+
+    describe("UCR", function() {
+      it("Create an ucr app with force-pull", function() {
+        const serviceName = "app-with-ucr-force-pull";
+        const container = "alpine";
+        const cmd = "while true; do sleep 100; done";
+
+        // Select 'Single Container'
+        cy.contains("Single Container").click();
+
+        // Fill-in the input elements
+        cy
+          .root()
+          .getFormGroupInputFor("Service ID *")
+          .type(`{selectall}{rightarrow}${serviceName}`);
+        cy.root().getFormGroupInputFor("Container Image").type(container);
+        cy.root().getFormGroupInputFor("Command").type(cmd);
+        // Select ucr runtime
+        selectUniversalRuntime();
+        cy.contains("Force Pull Image On Launch").click();
+
+        //
+        // TODO: Due to a bug in cypress you cannot type values with dots
```

Comment:
I see we're copying this sting over and over. If it's a real TODO let's file an issue at least.
Shall we also remove this comment from here?